### PR TITLE
Feature/wlenthe/citations

### PR DESCRIPTION
**Pull request by wlenthe**
created new branch to hold changes for pull request #428

### CODE REVIEW

File: Source/Plugins/EMMPM/EMMPMFilters/EMMPMFilter.cpp [hunk 1]
```diff
@@ -194,6 +194,8 @@ void EMMPMFilter::dataCheck()
     QString ss = QObject::tr("The minimum number of classes is 2");
     notifyErrorMessage(getHumanLabel(), ss, getErrorCondition());
   }
+
+  notifyWarningMessage(getHumanLabel(), "this filter contains references (a BibTeX file can be exported with 'Pipeline->Export Pipeline Citations...')", 0);
```
Comment (sdonegan) [inline Source/Plugins/EMMPM/EMMPMFilters/EMMPMFilter.cpp]: Capitalize "this" and "filter" for consistency

**Comment by sdonegan** [comment]
We all generally agree that this functionality is a good idea, and I think this approach is reasonable.  Only thing that leaves me somewhat uneasy is the "partial" implementation at this point; I think there are other filters out there that have references (definitely some based on the documentation), and I don't want this feature to come out and have it be confusing as to the source of the filter algorithm (i.e., someone sees this feature and assumes that if the produced BibTex for a given filter is empty that it has no citations, when it may in fact have some).

Basically, if it gets a citation in the documentation, it should get it in this BibTex implementation.  I'm willing to spend the time getting the list of filters and their citations together that need this put in if you can then add the BibTex to the filers themselves as part of this pull request.  I can probably get the time to knock this out this weekend.

**Comment by imikejackson** [comment]
Closing this pull request due to inability to obtain a CLA. The pull request may be reopened if a CLA is obtained in the future.
